GVFS/GVFS/CommandLine/MountVerb.cs

```diff
@@ -114,35 +114,27 @@ protected override void Execute(GVFSEnlistment enlistment)
 
                 if (!GVFSPlatform.Instance.KernelDriver.IsReady(tracer, enlistment.EnlistmentRoot, this.Output, out errorMessage))
                 {
-                    if (GVFSPlatform.Instance.UnderConstruction.SupportsGVFSService)
-                    {
-                        tracer.RelatedEvent(
-                            EventLevel.Informational,
-                            $"{nameof(MountVerb)}_{nameof(this.Execute)}_EnablingKernelDriverViaService",
-                            new EventMetadata
-                            {
-                                { "KernelDriver.IsReady_Error", errorMessage },
-                                { TracingConstants.MessageKey.InfoMessage, "Service will retry" }
-                            });
+                    tracer.RelatedEvent(
+                        EventLevel.Informational,
+                        $"{nameof(MountVerb)}_{nameof(this.Execute)}_EnablingKernelDriverViaService",
+                        new EventMetadata
+                        {
+                            { "KernelDriver.IsReady_Error", errorMessage },
+                            { TracingConstants.MessageKey.InfoMessage, "Service will retry" }
+                        });
 
+                    if (GVFSPlatform.Instance.ServiceSupportsDriverLoading())
+                    {
                         if (!this.ShowStatusWhileRunning(
-                            () => { return this.TryEnableAndAttachPrjFltThroughService(enlistment.EnlistmentRoot, out errorMessage); },
-                            $"Attaching ProjFS to volume"))
-                        {
-                            this.ReportErrorAndExit(tracer, ReturnCode.FilterError, errorMessage);
-                        }
+                        () => { return this.TryEnableAndAttachPrjFltThroughService(enlistment.EnlistmentRoot, out errorMessage); },

```

Comment:
This is probably for a follow up PR but this should be renamed to something like `TryLoadDriverThroughService` so that it isn't so Windows and PrjFlt specific.